apps/molgenis-components/src/components/layout/MolgenisSession.vue

```diff
@@ -57,7 +57,7 @@ import MolgenisAccount from "./MolgenisAccount.vue";
 import { request } from "../../client/client.js";
 
 const query = `{
-  _session { email, roles, schemas },
+  _session { email, roles, schemas, token, settings{key,value} },
```

Comment:
always al settings ?

Reply:
that is a good question. For now I don't know what to filter on so yes.